drydock_provisioner/drivers/node/maasdriver/models/iprange.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import drydock_provisioner.drivers.node.maasdriver.models.base as model_base
+
+class IpRange(model_base.ResourceBase):
+
+    resource_url = 'iprange/{resource_id}/'
+    fields = ['resource_id', 'comment', 'subnet', 'type', 'start_ip', 'end_ip']
+    json_fields = ['comment','start_ip', 'end_ip']
+
+    def __init__(self, api_client, **kwargs):
+        super(IpRange, self).__init__(api_client, **kwargs)
```

Comment:
The syntax for `super` has gotten simpler.  Now you can avoid repeating the class and `self`:

```python
super().__init__(api_client, **kwargs)
```